src/test/java/core/basesyntax/resources/input/three-fruits.csv

```diff
@@ -0,0 +1,14 @@
+type,fruit,quantity

```

Comment:
where should our resources directory be located?
same for the one in the `main`